src/WebAPI/appsettings.Development.json

```diff
@@ -0,0 +1,5 @@
+﻿{
+  "AllowedCORSOrigins": [
+    "https://localhost:5002"
+  ]
```

Comment:
Isn't the format comma-separated strings?

Reply:
Yeah, good catch. I've updated it